src/Microsoft.ApiDesignGuidelines.Analyzers/Core/PropertyNamesShouldNotMatchGetMethods.cs

```diff
@@ -15,35 +15,69 @@ public abstract class PropertyNamesShouldNotMatchGetMethodsAnalyzer : Diagnostic
         internal const string RuleId = "CA1721";
```

Comment:
Can you make this classes sealed and add the `[DiagnsoticAnalyzer(LanguageNames.CSharp, LanguageNames.VisualBasic)]` attribute to it and then delete the stubs for the c#\vb specific types and the fixers.